docs/UserGuide.md

```diff
@@ -157,13 +160,14 @@ For the good majority of visual learners, we figured to present information in a
 
 4. Double-click the file to start the app. The GUI similar to the below image should appear in a few seconds. Note how the app contains some sample data.<br>
    ![StartPage](images/StartPage.png)
+   <p align="center"> <sub> <b>Figure 3.1</b>: Example of start page </sub> </p>
 
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * **`liststu`** : Lists out all the students.
 
-   * **`addstu n/John Cena m/A0123456X e/thechamp@u.nus.edu`** : Adds a student named `John Cena` to the student list.
+   * **`addstu n/Jay Chou m/A0123456X e/jaychou@u.nus.edu`** : Adds a student named `Jay Chou` to the student list.
```

Comment:
Haha I thought we using Elvin Ng? But I also made changes to this. Will push later.

Reply:
Alright no problem!